### PR TITLE
Changed formula anttweakbar to include static lib

### PR DESCRIPTION
**Pull request by Morasta**
Added static lib install directive and updated Makefile.osx build flags for objective-c++.

### CODE REVIEW

**Comment by jacknagel** [comment]
This doesn't apply cleanly.

File: Library/Formula/anttweakbar.rb [hunk 1]
```diff
@@ -64,3 +63,16 @@ def install
          }
          if (gl_dyld)
          {
+diff --git a/src/Makefile.osx b/src/Makefile.osx
```
Comment (mikemcquaid) [inline Library/Formula/anttweakbar.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Comment (Morasta) [inline Library/Formula/anttweakbar.rb]: What's the best way to submit just this patch to the upstream developers (I think that's what you requested)?

Comment (mikemcquaid) [inline Library/Formula/anttweakbar.rb]: Probably create a ticket in their issue tracker, describe the problem, describe how this fixes the problem, link to the ticket as a comment above with an explanation of why Homebrew needs it. Thanks!

**Pull request closed without merging**